scala-package/core/src/main/scala/org/apache/mxnet/api/java/JavaNDArray.scala

```diff
@@ -1,95 +1,185 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 package org.apache.mxnet.api.java
 
 import org.apache.mxnet.NDArray
-import collection.JavaConverters
-import collection.JavaConverters._
-
-class JavaNDArray(val ndArray: NDArray) {
+import org.apache.mxnet.api.java.JavaDType.DType
 
-  /*
-  def this() = this(new NDArray())
-  (private[mxnet] val handle: NDArrayHandle,
-  val writable: Boolean = true,
-  addToCollector: Boolean = true)
-  */
-}
-
-object JavaNDArray{
+@AddJNDArrayAPIs(false)
+object JavaNDArray {
   implicit def fromNDArray(nd: NDArray): JavaNDArray = new JavaNDArray(nd)
 
-  implicit def toNDArray(jnd: JavaNDArray): NDArray = jnd.ndArray
-
-  val api = NDArray.api
+  implicit def toNDArray(jnd: JavaNDArray): NDArray = jnd.nd
 
   def waitall(): Unit = NDArray.waitall()
 
-  def onehotEncode(indices: JavaNDArray, out: JavaNDArray): JavaNDArray = NDArray.onehotEncode(indices, out)
+  def onehotEncode(indices: JavaNDArray, out: JavaNDArray): JavaNDArray
+  = NDArray.onehotEncode(indices, out)
```

Comment:
Don't make changes like this. The line was less than 120 to begin with so there's really not a good reason to split it. On top of that it makes doing PRs more complicated.

Reply:
The scalastyle limit the length as 100 instead of 120, so it is breaking scala style